spark/src/main/scala/io/substrait/spark/logical/ToSubstraitRel.scala

```diff
@@ -501,6 +503,12 @@ class ToSubstraitRel extends AbstractLogicalPlanVisitor with Logging {
               s"******* Unable to convert the plan to a substrait relation: " +
                 s"${logicalRelation.relation.toString}")
         }
+      case _: OneRowRelation =>
+        relation.EmptyScan
```

Comment:
EmptyScan doesn't produce any rows, so it's not an exact match for OneRowRelation, no? Would it make sense to convert it to VirtualTableScan with a single (empty) row in it instead?